DEBUG 2022-04-13 15:12:41 __init__.py: 278] CACHEDIR=/home/d/.cache/matplotlib
DEBUG 2022-04-13 15:12:41 font_manager.py: 1348] Using fontManager instance from /home/d/.cache/matplotlib/fontlist-v310.json
DEBUG 2022-04-13 16:58:40 __init__.py: 278] CONFIGDIR=/home/d/.config/matplotlib
DEBUG 2022-04-13 16:58:40 __init__.py: 278] (private) matplotlib data path: /mnt/hdd/PycharmProjects/venv/lib/python3.6/site-packages/matplotlib/mpl-data
DEBUG 2022-04-13 16:58:40 __init__.py: 278] matplotlib data path: /mnt/hdd/PycharmProjects/venv/lib/python3.6/site-packages/matplotlib/mpl-data
DEBUG 2022-04-13 16:58:40 __init__.py: 1009] loaded rc file /mnt/hdd/PycharmProjects/venv/lib/python3.6/site-packages/matplotlib/mpl-data/matplotlibrc
DEBUG 2022-04-13 16:58:40 __init__.py: 1598] matplotlib version 3.2.2
DEBUG 2022-04-13 16:58:40 __init__.py: 1599] interactive is False
DEBUG 2022-04-13 16:58:40 __init__.py: 1600] platform is linux
